src/config-hardware/displays.md

```diff
@@ -0,0 +1,77 @@
+# Configuring External Displays
+
+Navigate to `Settings` ➞ `Displays` to adjust multi-monitor placement, resolution, and scaling.
+
+![Display Settings Menu](/images/config-displays/display-settings.png)
```

Comment:
Good catch. I'll add a multi-monitor procedure section when I'm back at my desktop (I'm sure this could be done in a VM with some configuration, but for this I'd like to replicate a user's environment as closely as possible).

Reply:
I completed this, but I need to rescue some screenshots from my desktop which is suddenly booting to black on my Pop partition.